### DISCUSSION TITLE
Home Assistant integration

### DISCUSSION THREAD

**Discussion by lukyjay** [blinkospace/blinko#366] [Ideas]
Hi

Are there any plans to integrate with Home Assistant so that lists can be edited or displayed in Home Assistant?

**Comment by blinko-space** [comment]
The fact that we're a note-taking app. Are there a lot of people who need that?

**Reply by lukyjay** [reply]
> The fact that we're a note-taking app. Are there a lot of people who need that?

There are a lot of other note apps with integrations. Some here: https://www.home-assistant.io/integrations/#to-do-list and more in HACS

It's useful because:
1. You can link your existing links in the Lovelace interface (e.g. I have my shopping and to-do lists on my homepage)
2. Home Assistant has had a large push for local voice control and has just this week released a smart speaker, giving people local cloud-free voice access to managing lists
3. Users can use the integration to control their Blinko lists indirectly without needing to understand any APIs (e.g. home assistant will expose the lists as entities which can then be accessed via Node-Red, etc.)

Hope this helps! Sorry I didn't clarify the rationale in my original post 🙂

**Reply by blinko-space** [reply]
Oh, I understand what you mean ,it's a great idea. will consider adding this feature in a future release

**Reply by lukyjay** [reply]
Thank you and merry Christmas

**Reply by Charlielx** [reply]
I 100% should have read more before installing lol, because I literally set this up to replace Google Keep since Keep doesn't integrate with HA. This looks like an amazing service but HA integration is currently a must have for me, would love to see it added even if it was just very basic for now. Will keep checking this out though

**Reply by johnstan** [reply]
@Charlielx I've had Keep integrated with my HA for about a year now... there's a HACS custom integration ("Google Keep Sync") that works really well

**Comment by DeeDubs7** [comment]
I would love to see this as well

**Reply by wsw70** [reply]
I would love that, maybe as a plugin that sends notes)blinks via MQTT?
